packages/semantic-ui-theme/semantic/src/themes/sharegate/globals/site.overrides

```diff
@@ -1,3 +1,7 @@
 /*******************************
     Site Overrides
 *******************************/
+.ui,
+.o-ui {
+    color: var(--marine-500);
+}
```

Comment:
I'm fine if we keep it like this, but I want to point up the fact that we are adding this to SUI theme and for `o-ui` aren't SUI related.

Also we will at some point not use the SUI theme anymore.

Reply:
This is a valid point, where would you put it?

Reply:
I don't have the answer yet.